source/jormungandr/tests/authentication_tests.py

```diff
@@ -551,7 +551,7 @@ def test_places_authentication(self):
         # but when querying /v1/coverage/<something>/places only one pt_dataset is given to bragi
         with user_set(app, FakeUserAuth, 'super_user'):
             with requests_mock.Mocker() as m:
-                m.get(requests_mock.ANY, json=response)
+                m.post(requests_mock.ANY, json=response)
```

Comment:
Why is it a post now ?

Reply:
we use POST to send a shape in the body of the request